app/models/claim.rb

```diff
@@ -136,6 +137,12 @@ class Claim < ApplicationRecord
   has_one :support_ticket, dependent: :destroy
 
   belongs_to :eligibility, polymorphic: true, inverse_of: :claim, dependent: :destroy
+
+  belongs_to :journeys_session,
```

Comment:
Can we rename it to `journey_session` as `journeys_session` is an artifact of the namespacing and doesn't make sense in a singular context.